.travis.yml

```diff
@@ -40,3 +46,7 @@ deploy:
   on:
     branch: master
     condition: $TEST_SUITE = distribution
+
+notifications:
+  on_success: change
+  on_failure: always
```

Comment:
Are these not the defaults?

Reply:
Could be. Is it had to make them explicit?

Reply:
No, I just usually leave defaults alone as it keeps things smaller and easier to focus on what is actually being done.